server/src/TUManager.cpp

```diff
@@ -48,9 +48,15 @@ TUManager::TUManager()
   // significantly slower.
   //
   // -- https://github.com/Sarcasm/irony-mode/issues/4
-  if (CINDEX_VERSION < 6) {
-    parseTUOptions_ &= ~CXTranslationUnit_PrecompiledPreamble;
-  }
+#if CINDEX_VERSION < 6
```

Comment:
The previous version was intentional.
Generally speaking I prefer to avoid the preprocessor, because if there is a typo you don't know it unless you build in this configuration. `CXTranslationUnit_PrecompiledPreamble` has always existed to my knowledge it was just "broken" in an older versions.